Resources/Prototypes/Entities/Structures/Machines/frame.yml

```diff
@@ -105,3 +105,55 @@
     - type: Appearance
       visuals:
       - type: MachineFrameVisualizer
+
+
+- type: entity
+  id: MachineFrameDestroyed
+  name: destroyed machine frame
```

Comment:
IMO should inherit from the BaseDynamicStructure or whatever it's called and be wrenchable (via anchorablecomponent) etc.